diff --git a/L/LibCURL/common.jl b/L/LibCURL/common.jl
index abe2a57..5c26cb1 100644
--- a/L/LibCURL/common.jl
+++ b/L/LibCURL/common.jl
@@ -85,7 +85,7 @@ function build_libcurl(ARGS, name::String, version::VersionNumber; with_zstd=fal
         --without-brotli
 
         # A few things we actually enable
-	--with-libssh2=${prefix} --with-zlib=${prefix} --with-nghttp2=${prefix}
+        --with-zlib=${prefix} --with-nghttp2=${prefix}
         --enable-versioned-symbols
     )
 
@@ -118,7 +118,7 @@ function build_libcurl(ARGS, name::String, version::VersionNumber; with_zstd=fal
         FLAGS+=(--with-openssl)
     fi
 
-    if false; then
+    if [[ "${target}" == *linux* ]] || [[ "${target}" == *-freebsd* ]]; then
         # Use gssapi on Linux and FreeBSD
         FLAGS+=(--with-gssapi=${prefix})
         if [[ "${target}" == *-freebsd* ]]; then
@@ -175,6 +175,7 @@ function build_libcurl(ARGS, name::String, version::VersionNumber; with_zstd=fal
         Dependency("Zlib_jll"),
         Dependency("nghttp2_jll"),
         Dependency("OpenSSL_jll"; compat="3.0.16", platforms=openssl_platforms),
+        Dependency("Kerberos_krb5_jll"; platforms=filter(p->Sys.islinux(p) || Sys.isfreebsd(p), platforms)),
         BuildDependency(PackageSpec(name="LLVMCompilerRT_jll", uuid="4e17d02c-6bf5-513e-be62-445f41c75a11", version=llvm_version);
                         platforms=filter(p -> sanitize(p)=="memory", platforms)),
     ]
