diff --git a/L/LibCURL/common.jl b/L/LibCURL/common.jl
index b4391ff..df12641 100644
--- a/L/LibCURL/common.jl
+++ b/L/LibCURL/common.jl
@@ -53,7 +53,8 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
         --without-brotli
 
         # A few things we actually enable
-        --with-libssh2=${prefix} --with-zlib=${prefix} --with-nghttp2=${prefix}
+        # --with-libssh2=${prefix} --with-zlib=${prefix} --with-nghttp2=${prefix}
+        --with-zlib=${prefix} --with-nghttp2=${prefix}
         --enable-versioned-symbols
     )
 
@@ -64,7 +65,7 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
 
     if [[ ${target} == *mingw* ]]; then
         # We need to tell it where to find libssh2 on windows
-        FLAGS+=(LDFLAGS="${LDFLAGS} -L${prefix}/bin")
+        # FLAGS+=(LDFLAGS="${LDFLAGS} -L${prefix}/bin")
 
         # We also need to tell it to link against schannel (native TLS library)
         FLAGS+=(--with-schannel)
@@ -82,7 +83,8 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
         FLAGS+=(--with-openssl)
     fi
 
-    if false; then
+    # if false; then
+    if [[ "${target}" == *linux* ]] || [[ "${target}" == *-freebsd* ]]; then
         # Use gssapi on Linux and FreeBSD
         FLAGS+=(--with-gssapi=${prefix})
         if [[ "${target}" == *-freebsd* ]]; then
@@ -111,7 +113,7 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
     # These are the platforms we will build for by default, unless further
     # platforms are passed in on the command line
     platforms = supported_platforms()
-    push!(platforms, Platform("x86_64", "linux"; sanitize="memory"))
+    # push!(platforms, Platform("x86_64", "linux"; sanitize="memory"))
     # The products that we will ensure are always built
     if this_is_curl_jll
         # CURL_jll only provides the executable
@@ -129,11 +131,11 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
 
     # Dependencies that must be installed before this package can be built
     dependencies = [
-        Dependency("LibSSH2_jll"),
+        # Dependency("LibSSH2_jll"),
         Dependency("Zlib_jll"),
         Dependency("nghttp2_jll"),
         Dependency("OpenSSL_jll"; compat="3.0.15", platforms=filter(p->Sys.islinux(p) || Sys.isfreebsd(p), platforms)),
-        # Dependency("Kerberos_krb5_jll"; platforms=filter(p->Sys.islinux(p) || Sys.isfreebsd(p), platforms)),
+        Dependency("Kerberos_krb5_jll"; platforms=filter(p->Sys.islinux(p) || Sys.isfreebsd(p), platforms)),
         BuildDependency(PackageSpec(name="LLVMCompilerRT_jll", uuid="4e17d02c-6bf5-513e-be62-445f41c75a11", version=llvm_version);
                         platforms=filter(p -> sanitize(p)=="memory", platforms)),
     ]
